lib/phoenix_live_view/test/live_view_test.ex

```diff
@@ -1520,7 +1528,11 @@ defmodule Phoenix.LiveViewTest do
 
```

Comment:
Add the doc here for timeout as well ^^

Reply:
7e03a27e4b2bab6bbf8aedea1983849fcb87f633 👍